libs/cardano-ledger-core/src/Cardano/Ledger/MemoBytes.hs

```diff
@@ -60,48 +65,85 @@ import Prelude hiding (span)
 --   that were used to transmit it. Important since hashes are computed
 --   from the serialization of a type, and ToCBOR instances do not have unique
 --   serializations.
-data MemoBytes t = Memo {memotype :: !t, memobytes :: ShortByteString}
-  deriving (NoThunks) via AllowThunksIn '["memobytes"] (MemoBytes t)
-
-deriving instance NFData t => NFData (MemoBytes t)
-
-deriving instance Generic (MemoBytes t)
-
-instance (Typeable t) => ToCBOR (MemoBytes t) where
-  toCBOR (Memo _ bytes) = encodePreEncoded (fromShort bytes)
-
-instance (Typeable t, FromCBOR (Annotator t)) => FromCBOR (Annotator (MemoBytes t)) where
+data MemoBytes t era = Memo'
+  { memoType :: !(t era),
+    memoByteString :: ShortByteString,
```

Comment:
The issue here is that there is also a function named `memoBytes` in the same file and the field was previously named `memobytes` in lower caps, which is inconsistent with the way we usually name fields. If I named the field `memoBytes`, I'll also have to rename the other function and then change all the references to that function as well. I even tried that, but there were a lot of files that were using that while the field was not really used anywhere else AFAIK.

Reply:
I see, that definitely makes sense. Since the accessor is  only used in one place then let's call it `memoShortByteString`. It is more appropriate, since there is big difference between `ByteString` and `ShortByteString`

Reply:
I changed all fields to use `mb*` prefix instead of `memo*` before I saw your reply. Since the field accessors aren't used in many places, it shouldn't really matter either way.

Reply:
> I changed all fields to use mb* prefix

That's an even better solution :wink: :+1: